Python/Tests/Core.UI/EnvironmentUITests.cs

```diff
@@ -57,9 +57,7 @@ public class EnvironmentUITests {
             return app.SelectDefaultInterpreter(
                 PythonPaths.Python37 ?? PythonPaths.Python37_x64 ??
                 PythonPaths.Python36 ?? PythonPaths.Python36_x64 ??
-                PythonPaths.Python35 ?? PythonPaths.Python35_x64 ??
-                PythonPaths.Python34 ?? PythonPaths.Python34_x64 ??
-                PythonPaths.Python33 ?? PythonPaths.Python33_x64
+                PythonPaths.Python35 ?? PythonPaths.Python35_x64
```

Comment:
This change (and the similar ones below) are probably better written as `PythonPaths.Versions.First()` (or whatever the right logic is - I don't remember right now, but it might be `.Last()` or `.First(p => p.IsCPython)`)

Reply:
Okay, saw the logic below and remembered - you'll want `PythonPaths.Versions.Last(p => p.Version > PythonLanguageVersion.Vxx)`

Reply:
I've discussed this with Hugues and we decided to keep this as is for the time being. I'll add it to my list of things to revisit when I fix the broken tests.